### ISSUE TITLE
[Feature request]

### ISSUE THREAD

**Issue by Quorafind** [scambier/obsidian-omnisearch#101]
**Is your feature request related to a problem? Please describe.**

Now, omnisearch is working with all plaintext files, can it work with HTML file? I always save html by using save page WE .

**Describe the solution you'd like**

Treat html files as Txt files.

**Describe alternatives you've considered**

Use grep to request files.

**Additional context**

No.

**Comment by scambier** [comment]
> can it work with HTML file?

Yes, but actually no. Omnisearch will index all "words" in html files, including tags and attributes. HTML files need to be indexed as a special case, like PDFs.
